browser/net/brave_ad_block_tp_network_delegate_helper_unittest.cc

```diff
@@ -1,7 +1,7 @@
-/* Copyright (c) 2019 The Brave Authors. All rights reserved.
- * This Source Code Form is subject to the terms of the Mozilla Public
- * License, v. 2.0. If a copy of the MPL was not distributed with this file,
- * You can obtain one at https://mozilla.org/MPL/2.0/. */
+/* Copyright (c) 2019 The Brave Software Team. Distributed under the MPL2
+ * license. This Source Code Form is subject to the terms of the Mozilla Public
```

Comment:
Is this new? I thought we just used "Brave Authors" following chromium.

Reply:
Not sure where these came from, tbh - I'll undo them